libs/utils/env.py

```diff
@@ -603,8 +604,13 @@ def _init_platform_smp(self):
             # Try loading frequencies using the cpufreq module
             for cluster_id in self.platform['clusters']:
                 core_id = self.platform['clusters'][cluster_id][0]
-                self.platform['freqs'][cluster_id] = \
-                    self.target.cpufreq.list_frequencies(core_id)
+                try:
+                    self.platform['freqs'][cluster_id] = \
+                        self.target.cpufreq.list_frequencies(core_id)
+                except TargetError:
```

Comment:
You mean that if specify "cpufreq" among the modules to be loaded, than we should raise in case one of these modules is not properly working?

Regarding the ABI checks I would not assume that x86 never reports frequencies, is that the case?

Regarding `scaling_driver`, where do we get this info? You mean `target.cupfreq.get_all_governors()`?

Consider that if the cpufreq module has been loaded, than we should get a KeyError exception.
While if `cpufreq.list_frequencies()` should fails that we can quite likely assume that OPPs are not expose... the only other possible issue is a communication error...